executor/admin_test.go

```diff
@@ -105,6 +105,28 @@ func (s *testSuite5) TestAdminCheckIndexInTemporaryMode(c *C) {
 	tk.MustExec("drop table if exists temporary_admin_checksum_table_with_index_test,temporary_admin_checksum_table_without_index_test;")
 }
 
+func (s *testSuite5) TestAdminCheckIndexInLocalTemporaryMode(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("set @@tidb_enable_noop_functions = 1")
+	tk.MustExec("drop table if exists local_temporary_admin_test;")
+	tk.MustExec("create temporary table local_temporary_admin_test (c1 int, c2 int, c3 int default 1, primary key (c1), index (c1), unique key(c2))")
+	tk.MustExec("insert local_temporary_admin_test (c1, c2) values (1,1), (2,2), (3,3);")
+	_, err := tk.Exec("admin check table local_temporary_admin_test;")
+	c.Assert(err.Error(), Equals, core.ErrOptOnTemporaryTable.GenWithStackByArgs("admin check table").Error())
+	tk.MustExec("drop table if exists temporary_admin_test;")
+
+	tk.MustExec("drop table if exists local_temporary_admin_checksum_table_with_index_test;")
+	tk.MustExec("drop table if exists local_temporary_admin_checksum_table_without_index_test;")
+	tk.MustExec("create temporary table local_temporary_admin_checksum_table_with_index_test (id int, count int, PRIMARY KEY(id), KEY(count))")
+	tk.MustExec("create temporary table local_temporary_admin_checksum_table_without_index_test (id int, count int, PRIMARY KEY(id))")
+	_, err = tk.Exec("admin checksum table local_temporary_admin_checksum_table_with_index_test;")
+	c.Assert(err.Error(), Equals, core.ErrOptOnTemporaryTable.GenWithStackByArgs("admin checksum table").Error())
+	_, err = tk.Exec("admin checksum table local_temporary_admin_checksum_table_without_index_test;")
+	c.Assert(err.Error(), Equals, core.ErrOptOnTemporaryTable.GenWithStackByArgs("admin checksum table").Error())
+	tk.MustExec("drop table if exists local_temporary_admin_checksum_table_with_index_test,local_temporary_admin_checksum_table_without_index_test;")
```

Comment:
The title says `admin check index`

I see `admin check table` and `admin checksum table` 
... but where is the `admin check index`? 
https://github.com/pingcap/parser/blob/dd9b189324ced1cafb2ed35abf18c9d2037a614f/parser.y#L9722

Reply:
sorry, I thought `check table` contains index,  I fixed the title.

Reply:
check index wiil be added in https://github.com/pingcap/tidb/pull/26465/files